ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -196,4 +196,7 @@ p2p_uplinks_qos_profile: < qos_profile_name >
 # Enable PTP on all infrastructure links | Optional
 uplink_ptp:
   enable: < boolean | default -> false >
+
+# Enable Multicast in the underlay on all p2p uplink interfaces and mlag l3 peer interface.
```

Comment:
We should add some more details on what we enable. In this case it is only sparse-mode on the uplinks, and since we don't set any RP, we only support SSM for now.
```suggestion
# Enable Multicast in the underlay on all p2p uplink interfaces and mlag l3 peer interface.
# Specifically PIM Sparse-Mode will be configured on all routed underlay interfaces.
# No other configuration is added, so the underlay will only support Source-Specific Multicast (SSM)
# The configuration is intended to be used as multicast underlay for EVPN OISM overlay
```

Reply:
Thank you for this suggestion, it's been committed!
I rebased the branch and addressed conflicts, and ready for a final review!